client/src/Home.test.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { HomeComponent } from './components/home/Home';
+import renderer from 'react-test-renderer';
+import * as FetchConfig from './utils/FetchConfig';
+import { runFakeTimers } from './utils/TestUtils';
+import { Header } from './Header';
+import { setIconOptions, Spinner } from '@fluentui/react';
+import { Home } from './Home';
+import { AppConfigModel } from './models/ConfigModel';
+import { GenericError } from './components/GenericError';
+
+// Disable icon warnings for tests as we don't register the icons for unit tests which causes warnings.
+// See: https://github.com/microsoft/fluentui/wiki/Using-icons#test-scenarios
+setIconOptions({
```

Comment:
we can get rid of this as with recent changes we have these setting as part of setupTests.js

Reply:
done